cni/network/network.go

```diff
@@ -217,6 +226,14 @@ func (plugin *NetPlugin) findMasterInterface(nwCfg *cni.NetworkConfig, subnetPre
 	subnetPrefixString := subnetPrefix.String()
 	interfaces, _ := net.Interfaces()
 	for _, iface := range interfaces {
+		if subnetPrefixString == "" {
```

Comment:
add a new function findMasterInterfaceByMac

Reply:
how would be branch to this vs the default function used by other paths?